packages/core/static/html/logout.html

```diff
@@ -0,0 +1,14 @@
+<!-- https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#logoutsource -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Need to disable pinch zooming here, as it might be integrated into webview on mobile

Reply:
this page will not be able to interact, just a loading indicator

Reply:
Ah, right... hmm, then that's ok